entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/BaseEntityRecordService.java

```diff
@@ -586,7 +586,7 @@ protected void processCountryReference(Organization org) {
       // replace wikidata country ids
       org.setCountryId(europeanaCountryId);
       // search reference
-      EntityRecord orgCountry = entityRecordRepository.findByEntityId(europeanaCountryId);
+      EntityRecord orgCountry = entityRecordRepository.findEntityRecord(europeanaCountryId);
```

Comment:
this still retrieves the whole record from the database, while we only need the entityId and the pref labels.